packages/kbn-es/src/utils/parse_es_log.ts

```diff
@@ -38,16 +49,38 @@ export function parseEsLog(data: string) {
 
     capture = regex.exec(data);
   } while (capture);
+
   return lines;
 }
 
-function colorForLevel(level: string) {
-  switch (level) {
-    case 'WARN':
-      return chalk.yellow;
-    case 'DEBUG':
-      return chalk.dim;
-  }
+function extractEsLog(capture: RegExpExecArray) {
+  const [, , level, location, message] = capture;
 
-  return chalk.reset;
+  return { level, location, message };
+}
+
+/**
+ * extract useful info about an es log line
+ */
+export function parseEsLog(data: string) {
+  const regex = /\[([0-9-T:,]+)\]\[([A-Z]+)\s?\]\[([A-Za-z0-9.]+)\s*\]\s?([\S\s]+?(?=$|\n\[))/g;
+
+  return parseLog(data, regex, extractEsLog);
+}
+
+function extractDockerLog(capture: RegExpExecArray) {
```

Comment:
Optional, but if we want to skip the JSON log format and re-use pattern parsing `-e ES_LOG__STYLE=file`  should cause stdout to match the non-docker methods.

Reply:
Docker ES is now outputting in file style, so `parseEsDockerLog` is unused. Docker logs have to go through the parent process instead of a pipe as well, otherwise some output is lost. So, we can't perform any extra parsing before output. Which makes this dead code. Is it best to remove it then instead of *maybe* using it the future?

Reply:
+1 on removing if it's not used, we'll have history preserved here if we want to open it up.

Alternatively if there's a workflow we can think of to make use of it via a flag, maybe we can include that.  Offhand I can't think of anything.